src/towncrier/newsfragments/430.misc

```diff
@@ -0,0 +1,2 @@
+We now use `Nox <https://nox.thea.codes/>`_ instead of Tox.
+You may have to adjust a few command lines, but overall contributing should now be more comfortable.
```

Comment:
.. more comfortable.

arguably :) 

I guess that for people who like tox and already have tox, this is not more comfortable.

Unless @altendky  is not -1 for this, I think this can be merged.

Since there is a report of speed improvements, I am +1 for this.

nox, tox, potayto, potahto ... still not paver to me :)